tests/block_tools.py

```diff
@@ -182,7 +182,6 @@ def __init__(
             updated_constants = updated_constants.replace(**const_dict)
         self.constants = updated_constants
 
-        self.refresh_parameter: PlotsRefreshParameter = PlotsRefreshParameter(batch_size=2)
```

Comment:
Dropping `BlockTools.refresh_parameter` and using the ones from `BlockTools.plot_manager` below is required now since we create new params with `dataclasses.replace`.